### PR TITLE
Create contact form 

### PR DESCRIPTION
**Pull request by ayebaishmo**
In this  branch we:
- Create contact from in HTML
- Design in CSS
- Update the README file 
- Fixed the lint errors

### CODE REVIEW

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @ayebaishmo @sevinchek, 

Good job so far Team 👍 !
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* Please check comments below and make the necessary changes 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -184,39 +184,46 @@ <h4 class="dark"><span class="figure-3"></span>Frameworks</h4>
...
+                  <form action="https://formspree.io/f/mleanral" method="POST">
+                    <ul>
+                      <li>
+                        <input class="inpt" type="text" placeholder="Full name" id="name" maxlength="30" name="name" required>
+                      </li>
+                      <li>
+                        <input class="inpt" type="email" placeholder="Email address" id="email" name="email" required>
+                      </li>
+                      <li>
+                        <textarea class="txt-a" name="message" id="message" placeholder="Enter text here" maxlength="500" required>
```
[the remark this answers is not in the document]
Comment (ayebaishmo) [reply index.html:201]: The lint error dose not allow the type textarea

[the remark this answers is not in the document]
Comment (Davidosky007) [reply index.html:201]: ok 👍

[the remark this answers is not in the document]
Comment (Davidosky007) [reply index.html:201]: The form still submits when no message is entered [Link](https://www.loom.com/share/b5c80a8bec5448dbb27fa51d9814da9b)

[the remark this answers is not in the document]
Comment (Davidosky007) [reply index.html:201]: The form still submits when no message is entered check this video [Link](https://www.loom.com/share/b5c80a8bec5448dbb27fa51d9814da9b)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @sevinchek @ayebaishmo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
* You are almost done 🙏 check the comments and make the necessary changes then submit for review
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (ayebaishmo) [reply index.html:201]: > 
> 
> The form still submits when no message is entered check this video [Link](https://www.loom.com/share/b5c80a8bec5448dbb27fa51d9814da9b)
There is space with in the message that is why it submits, Back space and try submitting again. It won't submit

**Pull request merged**